src/components/directory-tree-view.js

```diff
@@ -250,9 +248,13 @@ const DirectoryTreeView = ({ treeViewUID, mapData }) => {
                 }}
             >
                 {Array.isArray(node.children)
-                    ? node.children.map((child) =>
-                          renderTree(mapDataRef.current[child.elementUuid])
-                      )
+                    ? node.children
```

Comment:
The sorting will not work for rootDirectories then (the first TreeItem in this recursive function)... we must add this sorting code in directory-container when we loop over rootdirectories to render TreeViews.